recipes/clusterone/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "clusterone" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://paccanarolab.org/static_content/clusterone/cluster_one-{{ version }}.jar
+  sha256: 3344c954df388f5cfe9795e0db375ae7189334a1eacb5bd3121507b33765a16b
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  run:
+    - openjdk >=8
+
+test:
+  commands:
+    - clusterone --version
+
+about:
+  home: https://paccanarolab.org/cluster-one/
+  license: GPL-3.0
+  license_file: LICENSE
```

Comment:
I think you need something like:
```
license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
to pick up the license from the recipe directory.

Reply:
No that should not be needed :-)

Reply:
So it's ok as is now?